packages/boa/package.json

```
@@ -4,7 +4,7 @@
   "description": "Use Python modules seamlessly in Node.js",
   "main": "lib/index.js",
   "scripts": {
-    "clean": "sh tools/clean-python.sh",
+    "clean": "sh tools/clean-python.sh; make -C ./pybind11/ clean",
```

Keep this consistent with other scripts:

```suggestion
    "clean": "sh tools/clean-python.sh && make -C ./pybind11/ clean",
```